tests/test_dxh.py

```diff
@@ -364,7 +364,7 @@ def _unit_mesh_boundary_indicator_function(spatial_coordinate):
     )
 
 
-def _zero_vector(vector: dolfinx.la.Vector):
+def _zero_vector(vector):
```

Comment:
I (stupidly) added a DOLFINx-version-specific type annotation to the tests 🤦.

At least we're now testing against these kinds of bugs.